clarify Apache Maven vs individuals (#18)
OpenElements/maven-support-care: README.md

> # Support & Care for Apache Maven

Reviewer: you will need to add few `&trade;`
see https://github.com/OpenElements/website/pull/89

This repository is used to track the work of [Support & Care for Apache Maven](https://open-elements.com/support-care-maven/) as open and transparent as possible.

<img width="1720" alt="logo" src="https://github.com/OpenElements/maven-support-care/assets/9443847/875e83bf-26e1-4488-a1fb-44e5de639f78">

The **Support & Care for Maven** Initiative is a groundbreaking program to help secure the future of the [Apache Maven project](https://maven.apache.org/) through sustainable support and care.

> The initiative aims to strengthen the development and maintenance of Maven through funding of individuals involved in the project, thus ensuring its high quality and availability that the Java ecosystem needs.

Reviewer 2: @hboutemy I'm afraid I have to disagree with this change. Our goal is not to create funding for individuals. Our goal is to create / get funding for work done on the project. My suggestion:

```suggestion
The initiative aims to strengthen the development and maintenance of Maven through funding of needed contributions to the project, thus ensuring the high quality and availability of Apache Maven that the Java ecosystem needs.
```

Author: @hendrikebbers I love it, thank you

## Our Mission

Apache Maven is a critical tool in the Java ecosystem, used by millions of developers worldwide with >100 million downloads per year.
Despite its importance, Maven is maintained primarily on a volunteer basis with no fixed financial support.

> Our mission is to create a sustainable source of funding for invidivuals to enable continuous improvements and professional support for Maven.

Reviewer 2: Suggest based on my previews argument:

```suggestion
Our mission is to create a sustainable source of funding for highly needed work around the Apache Maven project and direct contribution to the project. Our goal is to enable continuous improvements and professional support for Maven.
```
